airbyte-integrations/connectors/source-microsoft-sharepoint/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "type": "STREAM",
+    "stream": {
+      "stream_descriptor": {
+        "name": "test_csv"
+      },
+      "stream_state": {
+        "history": {
+          "Test_folder/TestFile.csv": "2030-11-17T13:52:35.000000Z",
```

Comment:
Hopefully, this source won't exists in 2030 so that we don't have to update this tests

Reply:
Fixed